plugin/vim-gh-line.vim

```diff
@@ -48,14 +48,27 @@ if !exists('g:gh_cgit_url_pattern_sub')
     let g:gh_cgit_url_pattern_sub = []
 endif
 
-func! s:gh_line(action) range
+if !exists('g:git_remote')
+    let g:git_remote = ""
+endif
+
+func! s:gh_line(action, force_interactive) range
     " Get Line Number/s
     let lineNum = line('.')
     let fileName = resolve(expand('%:t'))
     let fileDir = resolve(expand("%:p:h"))
     let cdDir = "cd '" . fileDir . "'; "
 
-    let origin = system(cdDir . "git config --get remote.origin.url")
+    " try to find git remote
+    if a:force_interactive == 1 || g:git_remote == ""
+      let g:git_remote = s:find_git_remote(cdDir)
+    endif
+
+    if g:git_remote == ""
+      echom "vim-gh-line CAN NOT find git remote"
+      return
+    endif
+    let origin = system(cdDir . "git config --get remote." . g:git_remote . ".url")
```

Comment:
With this change, the variable name `origin` does not improve code readability. 
What do you think about renaming this variable to `remote`? The used remote does not have to be origin anymore and calling it `origin` can be misleading.

Reply:
agree 💯